chapters/ch30.md

```diff
@@ -1,51 +1,51 @@
 
---> _replace THIS LINE by your translation for the above line_
+Đường cong màu đỏ thể hiện sai số của tập phát triển thường cao hơn so với đường cong màu xanh thể hiện sai số trong tập huấn luyện. Vì vậy, hầu như không có cách nào để thêm dữ liệu sao cho đường cong sai số giảm xuống mức hiệu suất mong muốn khi ngay cả sai số trong huấn luyện cũng cao hơn mức mong muốn đó.
 
 
 > Examining both the dev error curve and the training error curve on the same plot allows us to more confidently extrapolate the dev error curve.
 
---> _replace THIS LINE by your translation for the above line_
+Kiểm tra [phân tích] cả đường cong sai số của tập phát triển lẫn đường cong sai số của tập huấn luyện trong cùng một đồ thị cho phép chúng ta tự tin có những suy đoán [phép ngoại suy] về đường cong của tập phát triển [lặp từ]. 
 
 
 > Suppose, for the sake of discussion, that the desired performance is our estimate of the optimal error rate. The figure above is then the standard “textbook” example of what a learning curve with high avoidable bias looks like: At the largest training set size—presumably corresponding to all the training data we have—there is a large gap between the training error and the desired performance, indicating large avoidable bias. Furthermore, the gap between the training and dev curves is small, indicating small variance.
 
---> _replace THIS LINE by your translation for the above line_
+Để thảo luận, giả sử hiệu suất mong muốn chính là ước tính cho tỉ lệ lỗi tối ưu. Đồ thị trên trở thành ví dụ chuẩn "sách giáo khoa" về một đồ thị quá trình học với độ chệch cao có thể tránh [avoidable bias?] sẽ trông như thế nào: Tại điểm tập huấn luyện có kích cỡ lớn nhất -[có lẽ] tương ứng với tất cả dữ liệu trong tập huấn luyện - có một khoảng cách lớn giữa sai số huấn luyện và hiệu suất mong muốn, dấu hiệu của độ chệch có thể tránh được[avoidable bias?]. Hơn nữa, tại điểm này, khoảng cách giữa đường cong huấn luyện và đường cong phát triển nhỏ, dấu hiệu của phương sai bé. [đoạn này khá lủng củng]
 
 
 > Previously, we were measuring training and dev set error only at the rightmost point of this plot, which corresponds to using all the available training data. Plotting the full learning curve gives us a more comprehensive picture of the algorithms’ performance on different training set sizes.
 
---> _replace THIS LINE by your translation for the above line_
+Trước đó, chúng ta chỉ đo sai số của tập huyến luyện và sai số của tập phát triển tại điểm ngoài cùng bên phải của đồ thị, tương ứng với việc sử dụng hết dữ liệu trong tập huấn luyện. Vẽ đầy đủ đồ thị quá trình học sẽ cho chúng ta một cái nhìn tổng thể hơn về hiệu suất của những thuật toán trên các kích cỡ khác nhau của tập huấn luyện. [dịch lại câu này]
```

Comment:
- "sai số ~của~ tập huyến luyện và sai số ~của~ tập phát triển", em thấy bỏ từ "của" cũng không ảnh hưởng đến ý nghĩa, giảm từ cho nó gọn nhẹ.
> sai số của tập huyến luyện và sai số của tập phát triển
> Đề xuất "`Biểu diễn` đầy đủ đồ thị quá trình học sẽ cho chúng ta một `bức tranh` tổng thể hơn về hiệu `quả` của những thuật toán trên `từng` kích cỡ của tập huấn luyện"
- ~Vẽ~ đầy đủ đồ thị quá trình học sẽ cho chúng ta một ~cái nhìn~ tổng thể hơn về hiệu ~suất~ của những thuật toán trên các kích cỡ ~khác nhau~ của tập huấn luyện